sdk/healthinsights/azure-healthinsights-radiologyinsights/README.md

```diff
@@ -140,6 +263,8 @@ see the Code of Conduct FAQ or contact <opencode@microsoft.com> with any
 additional questions or comments.
```

Comment:
> Health Insights Radiology Insights client library will raise exceptions defined in [Azure Core][azure_core].

line 233, [azure_core] missing in Links references.

Reply:
Added